src/app/App.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import BaseApp from 'next/app';
+import Head from 'next/head';
+import keyboardOnlyOutlines from 'keyboard-only-outlines';
+import ReactGA from 'react-ga';
+
+import seoImage from '../shared/media/images/seo.png';
+
+import '../shared/styles/index.css';
+
+const SEO_DATA = {
+    url: 'https://{project-domain}',
+    title: '{project-name}',
+    description: '{project-description}',
+    keywords: ['{project-keyword-1}', '{project-keyword-2}'],
+    image: { src: seoImage, width: 1200, height: 630 },
+};
+
+class App extends BaseApp {
+    static async getInitialProps({ Component, ctx }) {
```

Comment:
We can remove this method and only add when needed: [Opt-out of Automatic Prerendering](https://github.com/zeit/next.js/blob/master/errors/opt-out-automatic-prerendering.md)

Reply:
Maybe we should take this to a recipe?

Reply:
Done, created a recipe for automatic prerendering